kratos/input_output/vtk_output.cpp

```diff
@@ -404,9 +378,28 @@ void VtkOutput::ForceBigEndian(unsigned char *bytes)
 std::string VtkOutput::GetOutputFileName(const ModelPart &rModelPart)
 {
     int rank = rModelPart.GetCommunicator().MyPID();
-    int step = rModelPart.GetProcessInfo()[STEP];
-    std::string output_file_name = mrOutputSettings["folder_name"].GetString() +"/"+
-                                    rModelPart.Name() + "_" + std::to_string(rank) + "_" + std::to_string(step) + ".vtk";
+    std::string model_part_name;
+    if(rModelPart.IsSubModelPart())
+        model_part_name = rModelPart.GetParentModelPart()->Name() + "_" + rModelPart.Name();
```

Comment:
this does not work with subsubmodelparts

Reply:
any way we are not outputting subsubmodelparts. Look in `PrintOutput` function !